docs/standard/library-guidance/index.md

```diff
@@ -21,23 +21,23 @@ Aspects of high-quality open-source .NET libraries:
 > [!div class="nextstepaction"]
 > [Get Started](./get-started.md)
 
-## Recommendations
+## Types of recommendations
 
-With each article, there is a list of recommendations for your .NET library using **Do**, **Consider**, **Avoid**, and **Do not**. The wording of each recommendation indicates how strongly it should be followed.
+Articles in this guidance present recommendations of four types: **Do**, **Consider**, **Avoid**, and **Do not**. The type of recommendation indicates how strongly it should be followed.
 
-A **Do** recommendation is one that should almost always be followed:
+You should almost always follow a **Do** recommendation. For example:
 
 **✔️ DO** distribute your library using a NuGet package.
 
 On the other hand, **Consider** recommendations should generally be followed, but there are legitimate exceptions to the rule and you shouldn't feel bad about not following the guidance:
 
 **✔️ CONSIDER** using [SemVer 2.0.0](https://semver.org/) to version your NuGet package.
 
-**Avoid** recommendations are things that are generally not a good idea, but breaking the rule sometimes makes sense:
+**Avoid** recommendations mention things that are generally not a good idea, but breaking the rule sometimes makes sense:
```

Comment:
Very good change!! I didn't like "are things" previously, but couldn't think of a better replacement.